lms/templates/edxnotes/edxnotes.html

```diff
@@ -112,7 +111,7 @@ <h2 class="placeholder-title">${_('You have not made any notes in this course ye
     <%static:require_module module_name="js/edxnotes/views/page_factory" class_name="NotesPageFactory">
         NotesPageFactory({
             disabledTabs: ${disabled_tabs},
-            notes: ${notes},
+            notes: ${dump_js_escaped_json(notes, NoteJSONEncoder) | n},
```

Comment:
Thanks @muzaffaryousaf.  I haven't reviewed this closely, but you'll want to properly escape all of the `${}` expressions inside `NotesPageFactory`.  You can follow [these docs](https://github.com/edx/edx-documentation/blob/robrap/safe-templates/en_us/developers/source/safe_templates.rst) that are in the process of being reviewed.